### PR TITLE
[All Hosts] (ie) clarify non-significance of IE deprecation

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Fixes [2756](https://github.com/OfficeDev/office-js-docs-pr/issues/2756)

### CODE REVIEW

File: docs/overview/explore-with-script-lab.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Explore Office JavaScript API using Script Lab
 description: 'Use Script Lab to explore the Office JS API and to prototype functionality.'
-ms.date: 10/14/2020
+ms.date: 06/18/2020
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:4]:
```suggestion
ms.date: 06/18/2021
```

File: docs/testing/ie-11-testing.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 
 # Test your Office Add-in on Internet Explorer 11
 
+> [!IMPORTANT]
+> **Internet Explorer Still Used in Office Add-ins**
+>
+> Microsoft is ending support for Internet Explorer, but this doesn't significantly affect Office Add-ins. Some combinations of platforms and Office versions, including all one-time-purchase versions through Office 2019, will continue to use the webview control that comes with Internet Explorer 11 to host add-ins, as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). Moreover, support for these combinations, and hence for Internet Explorer, is still required for add-ins submitted to [AppSource](/office/dev/store/submit-to-appsource-via-partner-center). Two things *are* changing:
+>
+> - AppSource no longer tests add-ins in Office on the web using Internet Explorer as the browser. But AppSource still tests for combinations of platform and Office *desktop* versions that use Internet Explorer.
+> - The [Script Lab tool](../overview/explore-with-script-lab.md) will stop working in Internet Explorer sometime in 2021.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/ie-11-testing.md:16]: You seem to have a couple of instances of this. Maybe use an include file?

File: docs/overview/explore-with-script-lab.md [hunk 3]
```diff
@@ -52,18 +52,21 @@ You can import a snippet into Script Lab either by specifying the URL to the pub
...
 
-- Outlook 2013 or later on Windows
+- Microsoft 365 Office subscription
 - Outlook 2016 or later on Mac
 - Outlook on the web when using Chrome, Microsoft Edge, or Safari browsers
 
 For more details on Script Lab for Outlook, see the related [blog post](https://developer.microsoft.com/outlook/blogs/script-lab-now-supports-outlook/).
 
+> [!IMPORTANT]
+> Sometime in 2021, Script Lab will stop working on the combinations of platform and Office version that use Internet Explorer to host add-ins. This includes one-time purchase Office versions through Office 2019, and some older versions of Microsoft 365 (subscription) Office. (For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).) You'll need other platform and version combinations to explore and test the Office JavaScript Library APIs with Script Lab. But the behavior of these APIs isn't different in Internet Explorer, so this isn't really a weakness of Script Lab. Note that Office add-ins submitted to [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) must support the platform and version combinations that use Internet Explorer to host add-ins.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md:68]:
```suggestion
> Sometime in 2021, Script Lab will stop working on the combinations of platform and Office version that use Internet Explorer to host add-ins. This includes one-time purchase Office versions through Office 2019, and some older versions of Microsoft 365 (subscription) Office. (For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).) You'll need other platform and version combinations to explore and test the Office JavaScript Library APIs with Script Lab. But the behavior of these APIs isn't different in Internet Explorer, so this isn't really a weakness of Script Lab. Note that Office Add-ins submitted to [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) must support the platform and version combinations that use Internet Explorer to host add-ins.
```

File: docs/testing/ie-11-testing.md (hunk 2 above)
Comment (Rick-Kirkham) [reply docs/testing/ie-11-testing.md:16]: There are only 2 where it is exactly the same and it seemed to me that an include file was more trouble than it's worth when its only being used twice.

File: docs/testing/ie-11-testing.md [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
 description: 'Test your Office Add-in on Internet Explorer 11.'
-ms.date: 05/19/2021
+ms.date: 06/18/2021
 localization_priority: Normal
 ---
 
 # Test your Office Add-in on Internet Explorer 11
 
+> [!IMPORTANT]
+> **Internet Explorer Still Used in Office Add-ins**
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/ie-11-testing.md:11]:
```suggestion
> **Internet Explorer still used in Office Add-ins**
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
ok, then one more fix. Looks good.

**Pull request merged**